gradle.properties

```diff
@@ -17,4 +17,6 @@ mocha_teamcity_reporter_version=2.2.2
 source_map_support_version=0.5.3
 
 kotlin.incremental.multiplatform=true
-kotlin.native.ignoreDisabledTargets=true
\ No newline at end of file
+kotlin.native.ignoreDisabledTargets=true
+
+kotlin.js.compiler=both
```

Comment:
Could you please elaborate on the specifics of this change? It's okay to do it in person, not here, whatever suits you.

Do I get it right: with 1.4-M1 the library is built twice (using old BE and IR BE) and published as an aggregated artifact with a corresponding metadata. And for Kotlin 1.4, Kotlin plugin knows how to pick the right artifact (old or a new one) using the published metadata?

Reply:
You are right. We build and publish both `.jar` and `.klib` with the same name and metadata to help consumers choose which one to use